apps/frontend/src/pages/auth/libs/components/sign-in-form/sign-in-form.tsx

```diff
@@ -49,11 +51,15 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 					label="Password"
 					name="password"
 					onIconClick={handleTogglePasswordVisibility}
-					placeholder="*******"
+					placeholder="••••••"
 					type={isPasswordVisible ? "text" : "password"}
 				/>
 
-				<Button label="SIGN IN" type="submit" />
+				<Button
+					isDisabled={Object.keys(errors).length > NO_ERROR_INPUT_FIELD}
```

Comment:
Create a separate variable for that

Reply:
Did you mean like this?

`const isInputError = Object.keys(errors).length > NO_ERROR_INPUT_FIELD`